src/components/quiz-questions/KdQuestion.tsx

```diff
@@ -55,15 +56,16 @@ const KdQuestion: React.FC<KdQuestionProps> = ({ reactionType }) => {
         <div className={styles.inputFormContent}>
             <p>
                 Referencing the Equilibrium Concentration plot, what is the
-                binding affinity?
+                binding affinity? (K<sub>d</sub> = ?)
             </p>
-            <b>
-                K<sub>d</sub> = ?
-            </b>
-            <InputNumber
-                value={selectedAnswer || ""}
-                onChange={handleAnswerSelection}
-            />
+            <Flex gap={8} align="baseline" style={{ maxWidth: 230 }}>
+                <InputNumber
+                    value={selectedAnswer || ""}
+                    onChange={handleAnswerSelection}
+                    placeholder="Type approximate value..."
+                />
+                <span> {MICRO}M</span>
+            </Flex>
```

Comment:
This InputNumber should have a `label` element or a `aria-labelledby` attribute linking to the  `p` question element